sevntu-checks/src/test/java/com/github/sevntu/checkstyle/filters/SuppressionAnnotationFilterTest.java

```diff
@@ -0,0 +1,279 @@
+        "135:5: Missing a Javadoc comment.",
+        "139:10: Name 'FOO3' must match pattern '^[a-z][a-zA-Z0-9]*$'.",
+        "146: Expected an @return tag.",
+        "146:9: Name 'FOO2' must match pattern '^[a-z][a-zA-Z0-9]*$'.",
+        "146:25: Expected @param tag for 'C1'.",
+        "146:25: Name 'C1' must match pattern '^[a-z][a-zA-Z0-9]*$'.",
+        "153:9: Name 'C2' must match pattern '^[a-z][a-zA-Z0-9]*$'.",
+        "153:13: Missing a Javadoc comment.",
+        "153:24: Name 'FOO3' must match pattern '^[a-z][a-zA-Z0-9]*$'.",
+        "153:34: Name 'C3' must match pattern '^[a-z][a-zA-Z0-9]*$'.",
+        "162:10: Unused @param tag for 'it'.",
+        "169:7: Missing a Javadoc comment.",
+        "171:21: Name 'C5' must match pattern '^[a-z][a-zA-Z0-9]*$'.",
+    };
+
+    @Test
+    public void testNone()
+            throws Exception {
+        final DefaultConfiguration filterConfig = null;
+        final String[] suppressed = {
+        };
+        verifySuppressed(filterConfig, suppressed);
+    }
+
+    @Test
+    public void testDefault()
+            throws Exception {
+        final DefaultConfiguration filterConfig =
+                createFilterConfig(SuppressionAnnotationFilter.class);
+        final String[] suppressed = {
+        };
+        verifySuppressed(filterConfig, suppressed);
+    }
+
+    //com.github.sevntu.checkstyle.filters
+    @Test
+    public void testNamesParsing1() throws Exception {
+        final DefaultConfiguration filterConfig =
+                createFilterConfig(SuppressionAnnotationFilter.class);
+        filterConfig.addAttribute("annotationNames", "@Test1");
+        filterConfig.addAttribute("annotationNames", "@com.github.sevntu.checkstyle.filters.Test2");
+        filterConfig.addAttribute("annotationNames", "Test3");
+        filterConfig.addAttribute("annotationNames", "com.github.sevntu.checkstyle.filters.Test4");
+        filterConfig.addAttribute("annotationNames", "com.github.sevntu.checkstyle.checks.test.external.InputSuppressionAnnotation");
+        final String[] resulted = {
+            "14:11: Missing a Javadoc comment.",
+            "21:5: Missing a Javadoc comment.",
+            "22:5: Missing a Javadoc comment.",
+            "64:5: Missing a Javadoc comment.",
+            "64:10: Name 'FOO1' must match pattern '^[a-z][a-zA-Z0-9]*$'.",
+            "70:32: Name 'A7' must match pattern '^[a-z][a-zA-Z0-9]*$'.",
+            "125:9: Name 'B10' must match pattern '^[a-z][a-zA-Z0-9]*$'.",
+            "131:22: Name 'FOO2' must match pattern '^[a-z][a-zA-Z0-9]*$'.",
+            "146: Expected an @return tag.",
+            "146:9: Name 'FOO2' must match pattern '^[a-z][a-zA-Z0-9]*$'.",
+            "162:10: Unused @param tag for 'it'.",
+            "171:21: Name 'C5' must match pattern '^[a-z][a-zA-Z0-9]*$'.",
+        };
+        verify(createChecker(filterConfig),
+                getPath("external/InputSuppressionAnnotationtFilter.java"), resulted);
```

Comment:
new inputs shouldn't be in `external` folder. That is only for files used by inputs alone and not part of the tests.
Fix this for all new inputs.